FAQ.md

```diff
@@ -0,0 +1,13 @@
+# Frequent Questions
+
+## Q: What docker version should I use if I have Windows 10 Home?
+A: Use [Docker Toolbox] (https://docs.docker.com/toolbox/toolbox_install_windows/)
+
+## Q: How does is works?
+A: You just have to download the latest .exe.After the instalation, when you turn it on it runs a Virtual Machine which use "normal" docker.
```

Comment:
Minor typos: 

- "... the latest `.exe`. After ... ", quote `exe`, and add whitespace. 
- "which uses"